Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.h

```diff
@@ -141,7 +141,7 @@ class SOFA_COMPONENT_CONSTRAINT_LAGRANGIAN_SOLVER_API GenericConstraintSolver :
     void computeAndApplyMotionCorrection(const core::ConstraintParams* cParams, GenericConstraintSolver::MultiVecId res1, GenericConstraintSolver::MultiVecId res2) const;
     void applyMotionCorrection(
         const core::ConstraintParams* cParams,
-        const core::MultiVecCoordId xId,
+        const std::unique_ptr<core::MultiVecCoordId>& xId,
```

Comment:
change of API so breaking 😶‍🌫️

Reply:
The function is private. Do you a case where changing the signature of a private method can be breaking?

Reply:
hum if somebody inherits it ? (and override it)